### PR TITLE
Fix for Lights with no brightness supported_features

### PR DESCRIPTION
**Pull request by illuzn**
Inserts all_icons.blank for `bri` if the light.supported_features does not include SUPPORT_BRIGHTNESS (bitmask 1).

This prevents the otherwise nonsense result of 0% showing here instead.

Addresses #1623

### CODE REVIEW

**Review by edwardtfn — APPROVED** [review]

File: nspanel_blueprint.yaml [hunk 1]
```diff
@@ -8473,9 +8473,13 @@ action:
                                             btn_bri_txt: >  # Buttons value (brightness, temperature, etc.)
                                               {% if not entity.has_value %} 0
                                               {% elif entity.domain == "light" and entity.state == "on" %}
-                                                {% set entity_brightness = state_attr(entity.id, "brightness") | default(None) %}
-                                                {% set entity_brightness_index = entity_brightness/255 if is_number(entity_brightness) else 0 %}
-                                                {{ (entity_brightness_index*100) | round(0) }}%
+                                                {% if (entity.supported_features and 1) == 0 %}
+                                                  {{ all_icons.blank }}
```
[the remark this answers is not in the document]
Comment (illuzn) [reply nspanel_blueprint.yaml]: I was trying to force it to an empty string but the way HA processes templates caused the empty string to be truncated and it returns None instead.

File: nspanel_blueprint.yaml [hunk 2]
```diff
@@ -8473,9 +8473,13 @@ action:
                                             btn_bri_txt: >  # Buttons value (brightness, temperature, etc.)
                                               {% if not entity.has_value %} 0
                                               {% elif entity.domain == "light" and entity.state == "on" %}
-                                                {% set entity_brightness = state_attr(entity.id, "brightness") | default(None) %}
-                                                {% set entity_brightness_index = entity_brightness/255 if is_number(entity_brightness) else 0 %}
-                                                {{ (entity_brightness_index*100) | round(0) }}%
+                                                {% if (entity.supported_features and 1) == 0 %}
```
[the remark this answers is not in the document]
Comment (illuzn) [reply nspanel_blueprint.yaml]: There may need to be a test of supported_features for covers. For lights and fans an attribute of 0% is clearly off; however, a position of 0% makes sense for a cover (closed) does it not.

[the remark this answers is not in the document]
Comment (edwardtfn) [reply nspanel_blueprint.yaml]: You're right. 0% makes sense for covers, however, I believe earlier versions didn't shown this when a cover was 0%... I will take a look at old code, but if that was the case, I will use something similar.
And there is `supported_features` for covers.

Thanks a lot.

[the remark this answers is not in the document]
Comment (edwardtfn) [reply nspanel_blueprint.yaml]: In this case a string with empty spaces shoukd do the trick. The all_icons.blank is not an empty string, it contains the utf-8 `\uFFFF` which could be printed depending on the font in use.

**Pull request merged**